assets/src/scripts/monitoring-location/store/instantaneous-value-time-series-data.js

```diff
@@ -192,12 +193,9 @@ const retrieveCustomTimePeriodIVTimeSeries = function(siteno, parameterCd, perio
         return getTimeSeries({sites: [siteno], params: [parameterCd], period: period}).then(
             series => {
                 const collection = normalize(series, tsRequestKey);
-                const variables = Object.values(collection.variables);
-                const variableToDraw = find(variables, v =>  v.variableCode.value === parameterCd);
-                dispatch(Actions.addIVTimeSeriesCollection(collection));
+                dispatch(Actions.addIVTimeSeriesCollection(convertTemperatureSeriesAndAddToCollection(collection)));
```

Comment:
I think the call to convert... should be made in the Action (before line 357). That way it's not repeated and may be easier to test. Unless there is a situation where we wouldn't call this but I don't think that happens now.

Reply:
Good idea. fixed.